helm-chart/images/binderhub/binderhub_config.py

```diff
@@ -83,6 +83,9 @@ def get_value(key, default=None):
 if os.getenv('BUILD_NAMESPACE'):
     c.BinderHub.build_namespace = os.environ['BUILD_NAMESPACE']
 
+if c.BinderHub.auth_enabled:
```

Comment:
Can you add `and 'base_url' in c.BinderHub` here? `base_url` is not typically going to be defined.